web/styles/zulip.css

```diff
@@ -189,6 +189,7 @@ body {
     --color-search-shadow-wide: hsl(0deg 0% 0% / 25%);
     --color-search-shadow-tight: hsl(0deg 0% 0% / 10%);
     --color-search-dropdown-top-border: hsla(0deg 0% 0% / 10%);
+    --color-recent-view-loading-spinner: hsl(0deg 0% 27%);
```

Comment:
This should probably go next to `--color-zulip-logo-loading` -- it'll help us deduplicate them later if we stick with these colors.

(In general it's worth some effort to keep these organized -- just appending items at the bottom )